libs/domains/environment/src/lib/mocks/deployment-log-factory.mock.ts

```diff
@@ -0,0 +1,43 @@
+import { Chance } from 'chance'
+import { EnvironmentLogs } from 'qovery-typescript-axios'
+import { LogsType } from '@qovery/shared/enums'
+
+const chance = new Chance()
+
+export const deploymentLogFactoryMock = (howMany: number, withError?: boolean): EnvironmentLogs[] =>
+  Array.from({ length: howMany }).map(() => ({
```

Comment:
Old guys like me would have done a for(const i, i++) loop... That method is nice! I like it!